packages/core/src/infrastructure/database/migrations/2024-03-31T15:36:45Z-share-email.ts

```diff
@@ -0,0 +1,12 @@
+import { Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>) {
+  await db.schema
+    .alterTable('students')
+    .addColumn('share_email', 'boolean')
```

Comment:
I _generally_ like the idea of having a prefix that hints that a field is a `boolean`. What do you think of `allow_share_email` or something of that sort?

Reply:
Ah yeah that's definitely more clear. How about `allow_email_share`?

Reply:
Even better, let's go with that!